docpages/example_programs/interactions_and_components/private-messaging.md

```diff
@@ -0,0 +1,80 @@
+\page private-messaging Sending private messages
+
+Sometimes it's simply not enough to ping someone in a server with a message, and we get that. That's why you can private message people! This tutorial will cover how to make a command that will either message the author of the command or message a specified user! This is done with a command option! So, let's get into it!
```

Comment:
I think it's unnecessary to tell that it's done with a command option, how else would you choose a user with a command? But overall intro is good 👍

Reply:
I disagree. Yes, it's the only way you'd choose a user, but it's still nice and friendly to say this. This also may not be entirely obvious to someone who is Discord bots and/or D++.

Reply:
I'm absolutely nobody to tell you what to do

Reply:
> I'm absolutely nobody to tell you what to do

I know that! This is the point of reviews! I'm adding more information to inform you on why I made the choice to have that, to see if you still stand by your point or not. If you do then that's okay and we can see what other people think.

Reply:
> I disagree. Yes, it's the only way you'd choose a user, but it's still nice...

I agree with it

Reply:
After further discussions in discord, we've come to a mutual agreement that the whole `This is done with a command option! So, let's get into it!` is a bit long and not needed. It's good to get straight to the point after `This tutorial...` so it has been removed.